### DISCUSSION TITLE
Does Amrex support Robin boundary conditions on Embedded Boundaries?

### DISCUSSION THREAD

**Discussion by starwar9527** [AMReX-Codes/amrex#3390] [Q&A] [answered]
Hi, the documentation says _"AMReX supports multi-level solvers for use with embedded boundaries. These include 1) cell-centered solvers with homogeneous Neumann, homogeneous Dirichlet, or inhomogeneous Dirichlet boundary conditions on the EB faces, and 2) nodal solvers with homogeneous Neumann boundary conditions, or inflow velocity conditions on the EB faces."_

But it does not mention about Robin boundary condition. Does Amrex support Robin boundary conditions on Embedded Boundaries?

**Comment by WeiqunZhang** [comment, marked as answer]
The linear system solvers in amrex do not support Robin boundary conditions. However, you might be able to set up Hypre to do this using amrex::HypreSolver in amrex/Src/Extern/HYPRE. It requires you to fill the matrix explicitly.

**Reply by starwar9527** [reply]
thanks a lot for your reply!
